designs/NoSQL_note.md

```diff
@@ -0,0 +1,83 @@
+# NoSQL
+
+Due to the technical limitations of setting up SQL, we decide to take the NoSQL approach by utilizing DynamoDB table on AWS, which has a trivial entry.
+
+## Tables
+
+### Customers:
+ - id: `string`
+ - email: `string`
+ - address: `string`
+ - payment_method: `TODO`
+ - phone_number: `string`
+ - order_ids: `List[string]`
+
+#### Example:
+ - id: Thomas
+ - email: xxxxx@gmail.com
+ - address: Unit 123 23th street, Vancouver,BC
+ - payment_method: `TODO`
+ - phone_number: 236863xxxx
+ - order_ids: [123321, 1234321, 5321321, ...]
+
+ ### Orders
+ - id: `string`
+ - customer_id: `string`
+ - restaurant_id: `string`
+ - content : `list[string]`
+ - extra_charge: `string`
```

Comment:
This can be a number as it is just a numerical value. Then maybe the currency can be separated out as a different field.